src/polyfills/details/details.js

```diff
@@ -30,6 +30,11 @@ var componentName = "wb-details",
 
 		if ( summary ) {
 			details = summary.parentNode;
+			Object.defineProperty( details, "open", {
```

Comment:
Are we using this pattern anywhere else in the WET code?  It looks great, but it concerns me with the amount of pattern mixing the WET codebase already has that we would just start using a new pattern again.